code/datums/diseases/advance/symptoms/thermoregulation.dm

```diff
@@ -0,0 +1,39 @@
+/** Thermoregulation
+ * No change to stealth.
+ * Increases resistance.
+ * Reduces stage speed.
+ * Reduces transmissibility
+ * Bonus: Regulates body temperature.
+ */
+
+/datum/symptom/thermoregulation
+	name = "Thermoregulation"
+	desc = "The virus reacts to extreme conditions and assists the body in regulating its temperature."
+	stealth = 0
+	resistance = 1
+	stage_speed = -2
+	transmittable = -1
+	level = 5
+	severity = 0
+	threshold_descs = list(
+		"Resistance 8" = "Increases thermal regulation speed."
+	)
+
+/datum/symptom/thermoregulation/Start(datum/disease/advance/A)
+	. = ..()
+	if(!.)
+		return
+	if(A.totalResistance() >= 8)
+		power = 1.75
```

Comment:
so weird this is at the base level of the symptom makes me nervous but i didnt see anything sooooo